spec/request/apps_spec.rb

```diff
@@ -14,14 +15,14 @@
 
   describe 'POST /v3/apps' do
     it 'creates an app' do
-      buildpack      = VCAP::CloudController::Buildpack.make
+      buildpack      = VCAP::CloudController::Buildpack.make(stack: stack.name)
       create_request = {
         name: 'my_app',
         environment_variables: { open: 'source' },
         lifecycle: {
           type: 'buildpack',
           data: {
-            stack: nil,
+            stack: buildpack.stack,
```

Comment:
Can we have a test for the case where `nil` gets passed into the request as the stack name?

- @selzoc && maryam

Reply:
I would test these permutations at the controller level. Request specs and generally happy path.

Reply:
And I think that buildpacks_controller_spec has a `#create defaults stack to unknown when nil` at https://github.com/cloudfoundry/cloud_controller_ng/pull/1064/commits/49b0a6676be3bb5f24831f8eeedc462372c9c375#diff-77441f6f04beceda9b6bb6c88e5a0643R64